dev/bots/test.dart

```diff
@@ -851,8 +850,8 @@ Future<void> _runFrameworkTests() async {
 
   Future<void> runPrivateTests() async {
     final List<String> args = <String>[
-      'run',
       '--sound-null-safety',
```

Comment:
It seems to me that `dart run <option> x.dart` the `<option>` belongs to the application itself (and not other isolates). For example:
```
% cat hello.dart
main() {
  assert(false);
  print("hello world");
}
% out/ReleaseX64/dart-sdk/bin/dart run --enable-asserts hello.dart
Unhandled exception:
'hello.dart': Failed assertion: line 2 pos 10: 'false': is not true.
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
#2      main
```

It does also work to put `--enable-asserts` before `run` but that may mean it applies to all isolates - which we may not want.

/cc @bkonyi Could you advise here?